backend/src/test/java/net/pengcook/image/controller/S3ControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package net.pengcook.image.controller;
+
+import static com.epages.restdocs.apispec.RestAssuredRestDocumentationWrapper.document;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+import static org.springframework.restdocs.request.RequestDocumentation.parameterWithName;
+import static org.springframework.restdocs.request.RequestDocumentation.queryParameters;
+
+import io.restassured.RestAssured;
+import net.pengcook.RestDocsSetting;
+import net.pengcook.image.dto.PresignedUrlResponse;
+import net.pengcook.image.service.S3ClientService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+class S3ControllerTest extends RestDocsSetting {
+
+    @MockBean
+    private S3ClientService s3ClientService;
+
+    @Test
+    @DisplayName("presigned url을 요청한다.")
```

Comment:
팀 코드 컨벤션에서 성공하는 테스트의 DisplayName은 `{행위}가 된다.` 로 되어있어서, 서버 입장에서 행위 위주로 작성하는건 어떨까요?
```suggestion
    @DisplayName("presigned url을 응답한다.")
```

Reply:
오 그렇네요. 서버 입장으로 DisplayName 적기 👍👍

Reply:
반영하였습니다!